stdlib/StdlibModules

```diff
@@ -44,7 +44,7 @@ STDLIB_MODULE_BASENAMES = \
   fun gc digest bigarray random hashtbl weak \
   format scanf callback camlinternalOO oo camlinternalMod ephemeron \
   filename complex arrayLabels listLabels bytesLabels stringLabels moreLabels \
-  stdLabels in_channel out_channel effect
+  stdLabels in_channel out_channel effect type
```

Comment:
Just a nit since there's not much to say :-) `STDLIB_MODULE_BASENAME` is supposed to be in dependency order, I think it would be better to have `type` further up so that the day people want to use it somewhere else in the stdlib they don't get hampered.

Reply:
(If you have the mental fortitude to do it, don't hesitate to also add line breaks to this definition so that there is only one item per line. It's annoying when you do it first, but then generations of committers and reviewers are going to thank you for the absence of paragraph-filling worries and the clean diffs.)

Reply:
Done (for both points), thanks.